tools/c7n_gcp/tests/data/iam-permissions.json

```diff
@@ -983,6 +983,7 @@
     "compute.instances.getSerialPortOutput",
     "compute.instances.getShieldedInstanceIdentity",
     "compute.instances.getShieldedVmIdentity",
+    "compute.instances.insert",
```

Comment:
this doesn't appear to be valid, we're generating this file from the upstream reference docs (see tools/dev/gcpiamdb.py) 
note methods may not directly correspond to permissions, there's an implied mapping thats undocumented per se, though it generally is 1:1


https://cloud.google.com/iam/docs/custom-roles-permissions-support

Reply:
Permission itself is not valid :(
It should be `compute.machineImages.create`
But its checking it because its not taking into consideration https://github.com/cloud-custodian/cloud-custodian/pull/6265/files#diff-ca4c088266dea74eb1497d80e0ec514cf58ddfd7840440537309c95dfc5acd9dR189

If Im wrong and I can fix it somehow let me know.

Reply:
right, the method here `insert` doesn't correspond 1:1 to a permission, the actual permission appears to be `compute.images.create`